packages/kbn-apm-synthtrace/src/scenarios/other_transaction_group_bucket.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import { range as lodashRange } from 'lodash';
+import { ApmFields, apm } from '@kbn/apm-synthtrace-client';
+import { Scenario } from '../cli/scenario';
+
+const scenario: Scenario<ApmFields> = async (runOptions) => {
+  const { logger } = runOptions;
+  const numServices = 10;
+
+  return {
+    generate: ({ range }) => {
+      const TRANSACTION_TYPES = ['request'];
+      const ENVIRONMENTS = ['production', 'development'];
+
+      const MIN_DURATION = 10;
+      const MAX_DURATION = 1000;
+
+      const MAX_BUCKETS = 50;
+
+      const BUCKET_SIZE = (MAX_DURATION - MIN_DURATION) / MAX_BUCKETS;
+
+      const instances = lodashRange(0, numServices).flatMap((serviceId) => {
+        const serviceName = `service-${serviceId}`;
+
+        const services = ENVIRONMENTS.map((env) => apm.service(serviceName, env, 'go'));
+
+        return lodashRange(0, 2).flatMap((serviceNodeId) =>
+          services.map((service) => service.instance(`${serviceName}-${serviceNodeId}`))
+        );
+      });
+
+      const transactionGroupRange = [
+        ...lodashRange(0, 10).map((groupId) => `transaction-${groupId}`),
+        'other',
+      ];
+
+      return range
+        .interval('1s')
+        .rate(1)
```

Comment:
Shorthand: 
```suggestion
        .ratePerMinute(60)
```

Reply:
`ratePerMinute` doesn't exist, I'll change it to `rate(60)`

Reply:
I'm pretty sure it does. We also use it here: https://github.com/kpatticha/kibana/blob/cee670a1ec8714b6366e0a396d49f9162657c048/packages/kbn-apm-synthtrace/src/scenarios/aws_lambda.ts#L19

Reply:
Keep in mind that 1 document per second is different from 60 docs per minute.

Reply:
`ratePerMinute` is supposed to smooth it out so you don't have to think about this. I see it has been removed. Will open a PR to re-add that logic.

Reply:
it is still there. https://github.com/elastic/kibana/blob/61dd55fff3a11f645a59f920f4514bc717179f63/packages/kbn-apm-synthtrace-client/src/lib/timerange.ts#L18. What do you mean with "smooth it out so you don't have to think about this"?

Reply:
https://github.com/elastic/kibana/pull/148707